grouper/fe/templates/group-request-update.html

```diff
@@ -28,6 +29,9 @@
                     <td>{{ update.from_status|default("", True) }}</td>
                     <td>{{ update.to_status }}</td>
                     <td>{{ update.change_at|print_date }}</td>
+                    {% if request.changes.expiration is defined %}
```

Comment:
I think we always want the `<td>` to be there, even if there's no text in it. Also, can we add/update an itest for this?